admin/winstonLogger.js

```diff
@@ -0,0 +1,73 @@
+'use strict'
```

Comment:
this file is unnecessary, as we should be logging to the same source as configured in `admin/app.js`

Reply:
This is the new logger, created only for API calls. I need a way to identify this logs separately but I couldn't find a neat way to do it in the app.js implementation.